pkg/store/gcworker/gc_worker.go

```diff
@@ -1239,36 +1231,25 @@ func (w *GCWorker) resolveLocks(
 
 const gcOneRegionMaxBackoff = 20000
 
-func (w *GCWorker) uploadSafePointToPD(ctx context.Context, safePoint uint64) error {
-	var newSafePoint uint64
-	var err error
-
-	bo := tikv.NewBackofferWithVars(ctx, gcOneRegionMaxBackoff, nil)
-	for {
-		newSafePoint, err = w.pdClient.UpdateGCSafePoint(ctx, safePoint)
-		if err != nil {
-			if errors.Cause(err) == context.Canceled {
-				return errors.Trace(err)
-			}
-			err = bo.Backoff(tikv.BoPDRPC(), errors.Errorf("failed to upload safe point to PD, err: %v", err))
-			if err != nil {
-				return errors.Trace(err)
-			}
-			continue
-		}
-		break
+func (w *GCWorker) broadcastGCSafePoint(ctx context.Context, gcSafePoint uint64) error {
+	result, err := w.pdGCControllerClient.AdvanceGCSafePoint(ctx, gcSafePoint)
```

Comment:
Before we use keyspace level GC, there is only one gc worker leader invoking this kind of API, so the QPS should be low enough.
Now one PD may serve many keyspace, **so concurrency matters**.
In the PD side there is a big mutex to protect all things for this kind of API, 
we should do more test to verify the performance later.